pkg/util/system/system_utils.go

```diff
@@ -18,14 +18,25 @@ package system
 
 import (
 	"strings"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
```

Comment:
All right, thank you.
The "moving the new labels and the old ones below to pkg/api/types.go" from that PR, I will update my PR and serviceController.go after merging that PR.